ARCHITECTURE.md

```diff
@@ -74,7 +74,7 @@ Important modules to note are
   a random variable distribution from a likelihood distribution.
 
 * `logprob.py`: This contains the log probability logic for the distributions themselves.
-  The log probability calculation is deferred to Aesara
+  The log probability calculation is deferred to PyTensor
```

Comment:
I think this is at least partially incorrect as "non-basic" distributions have their PPLs in pymc (logprob module)